config.go

```diff
@@ -46,12 +53,20 @@ func NewConfigFromFile(r io.Reader) (*Config, error) {
 	if config.DefaultTimeout == 0 {
 		config.DefaultTimeout = DefaultTimeout
 	}
+	if config.DefaultHTTPClientTimeout == 0 {
+		config.DefaultHTTPClientTimeout = DefaultHTTPClientTimeout
+	}
 	for t := range config.Targets {
 		if config.Targets[t].Timeout == 0 {
 			target := config.Targets[t]
 			target.Timeout = config.DefaultTimeout
 			config.Targets[t] = target
 		}
+		if config.Targets[t].HTTPClientTimeout == 0 {
+			target := config.Targets[t]
+			target.HTTPClientTimeout = config.DefaultHTTPClientTimeout
+			config.Targets[t] = target
+		}
```

Comment:
```suggestion
		target := config.Targets[t]
		config.Targets[t] = target
		if config.Targets[t].Timeout == 0 {
			target.Timeout = config.DefaultTimeout
		}
		if config.Targets[t].HTTPClientTimeout == 0 {
			target.HTTPClientTimeout = config.DefaultHTTPClientTimeout
		}
```

Reply:
Hmmm, maybe. This could go horribly wrong, I'm only editing this in the browser. :sweat_smile: